add_to_app/android_view/android_view/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.2.0-beta02"
+        classpath 'com.android.tools.build:gradle:4.0.0'
```

Comment:
This was requiring people to use a beta version of android studio to build the sample.